### DISCUSSION TITLE
Github

### DISCUSSION THREAD

**Discussion by dickiedyce** [johnlindquist/kit#729] [Q&A] [answered]
Why do I get a prompt to:

_Open https://github.com/login/device, paste code from clipboard_

when trying to create a Gist?

Assumption: I need to set some ENV variables somewhere? Perhaps some docs on the assumptions for using ScriptKit might be useful? Or am I looking in the wrong place?

**Comment by johnlindquist** [comment, marked as answer]
Your assumption is correct. Clicking the link and pasting the code will automatically create the env variable required for creating gists.

I'll consider different messaging in the dialogue to clarify what it's doing.
